src/screens/SearchScreen.tsx

```diff
@@ -22,13 +30,15 @@ const SearchScreen = () => {
   }
 
   return (
-    <Container>
-      <View style={styles.searchBarContainer}>
-        <SearchBar handleTextChanges={handleTextChanges} />
+    <SafeAreaView style={styles.searchBarContainer}>
```

Comment:
What about Container?

Reply:
Can't use it here because this screen needs to access the complete screen and I wasn't update the color of safe areas when inside `<Container>`

Reply:
What do you mean by this?

Reply:
When I use `SafeAreaView` (React native) then I can change background color of status bar. But if I use `Container` which uses `SafeAreaView` (`react-native-safe-area-context`) you can’t change the color of status bar because we can’t go beyond safe area view of device and it keeps native device color.

![CC5A5466-D4D2-4016-9D1A-09D6130B1FBC](https://user-images.githubusercontent.com/44710980/93602219-f7b5a800-f987-11ea-997a-23122dee0047.jpeg)

Reply:
If you replace it with `Container` you will see the difference. (Don’t forget to use bright colors )